src/api.js

```diff
@@ -4,8 +4,10 @@ const newsAPI = axios.create({
 	baseURL: 'https://nc-news-xd69.onrender.com/api',
 });
 
-const fetchArticles = async (sortQueryString = 'created_at') => {
-	const results = await newsAPI.get(`/articles?sort_by=${sortQueryString}`);
+const fetchArticles = async (
+	defaultSortString = '?sort_by=created_at&order_by=desc'
+) => {
+	const results = await newsAPI.get(`/articles${defaultSortString}`);
```

Comment:
I would change this to make use of Axios params. If you don't pass in a sort_by or order_by, Axios will ignore them when composing the query string automatically.